sammi/cdf_attribute_manager.py

```diff
@@ -312,3 +538,89 @@ def get_variable_attributes(
                 output[attr_name] = ""
 
         return output
+
+    def variable_attribute_template(self) -> OrderedDict:
+        """
+        Function to generate a template of required variable attributes
+        that must be set for a valid data file.
+
+        Returns
+        -------
+        template: `OrderedDict`
+            A template for required variable attributes that must be provided.
+        """
+        template = OrderedDict()
+        for attr_name, attr_schema in self.variable_attribute_schema[
+            "attribute_key"
+        ].items():
+            if attr_schema["required"]:
+                template[attr_name] = None
+        return template
+
+    def variable_attribute_info(self, attribute_name: Optional[str] = None):
```

Comment:
Same comment as above about returning a dict. Also, this function is missing a return typehint